erpnext/education/doctype/student/student.py

```diff
@@ -52,7 +52,7 @@ def update_student_name_in_linked_doctype(self):
 	def check_unique(self):
 		"""Validates if the Student Applicant is Unique"""
 		student = frappe.db.sql("select name from `tabStudent` where student_applicant=%s and name!=%s", (self.student_applicant, self.name))
-		if student:
+		if student and student.student_email_id != frappe.session.user.email:
```

Comment:
Can you please add some comments to explain the reason for adding the condition

Reply:
this is to avoid throwing the error id the user is already registered and logined. user should be able to apply for different courses.